backend/api/Profiles/RoleProfile.cs

```diff
@@ -13,17 +14,26 @@ public RoleProfile()
         {
             CreateMap<Entity.Role, RoleModel>()
                 .IncludeBase<Entity.LookupEntity, CodeModel>()
-                .ForMember(src => src.Id, opt => opt.MapFrom(dest => dest.Id.ToString()))
-                .ForMember(src => src.Users, opt => opt.MapFrom(src => src.Users.Select(u => u.User)));
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id.ToString()))
+                .ForMember(dest => dest.Users, opt => opt.MapFrom(src => src.Users.Select(u => u.User)));
 
             CreateMap<Entity.Role, CodeModel>()
                 .IncludeBase<Entity.LookupEntity, CodeModel>()
-                .ForMember(src => src.Id, opt => opt.MapFrom(dest => dest.Id.ToString()))
-                .ForMember(src => src.Code, opt => opt.MapFrom(dest => dest.Name));
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id.ToString()))
+                .ForMember(dest => dest.Code, opt => opt.MapFrom(src => src.Name));
 
             CreateMap<RoleModel, Entity.Role>()
-                .ForMember(src => src.Id, opt => opt.MapFrom(dest => Guid.Parse(dest.Id)))
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => Guid.Parse(src.Id)))
                 .IncludeBase<CodeModel, Entity.LookupEntity>();
+
+            CreateMap<Entity.UserRole, RoleModel>()
```

Comment:
Had to add this to solve automapper errors occurring during initialization.

Reply:
thanks for this.